metadata-ingestion/src/datahub/ingestion/source/usage/usage_common.py

```diff
@@ -112,6 +119,7 @@ def make_usage_workunit(
 
 class BaseUsageConfig(BaseTimeWindowConfig):
     top_n_queries: pydantic.PositiveInt = 10
+    indent: bool = True
```

Comment:
Can we leave this defaulting to False until its further validated? Then we can flip it to True if all works well for some time

Reply:
I modified as you suggest.

Reply:
Thank you!